pkg/apis/triggers/v1alpha1/event_listener_validation_test.go

```diff
@@ -38,113 +37,276 @@ func Test_EventListenerValidate(t *testing.T) {
 		el   *v1alpha1.EventListener
 	}{{
 		name: "TriggerTemplate Does Not Exist",
-		el: bldr.EventListener("name", "namespace",
-			bldr.EventListenerSpec(
-				bldr.EventListenerTrigger("dne", "v1alpha1"))),
+		el: &v1alpha1.EventListener{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "name",
+				Namespace: "namespace",
+			},
+			Spec: v1alpha1.EventListenerSpec{
+				Triggers: []v1alpha1.EventListenerTrigger{{
+					Template: &v1alpha1.EventListenerTemplate{
+						Ref:        ptr.String("dne"),
+						APIVersion: "v1alpha1", // TODO: test with v1alpha1 version as well.
```

Comment:
Comment doesn't quite make sense (it looks like we're already testing v1alpha1?) - remove or rephrase?